src/community/china/_os-settings.md

```diff
@@ -0,0 +1,124 @@
+   {% capture envvarset -%}{{prompt}} {{comtoset}}{% endcapture -%}
+   {% capture setpath -%}{{envvarset}}PATH = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"{% endcapture -%}
+   {% capture newdir -%}{{prompt}} New-Item -Path '{{installdirsuggestion}}' -ItemType Directory{% endcapture -%}
+   {% capture unzip -%} {{prompt}} Extract-Archive:{% endcapture -%}
+   {% capture permaddexample -%}
+$newPath = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"
+[System.Environment]::SetEnvironmentVariable('Path',$newPath,User)
+[System.Environment]::SetEnvironmentVariable('PUB_HOSTED_URL','https://pub.flutter-io.cn',User)
+[System.Environment]::SetEnvironmentVariable('FLUTTER_STORAGE_BASE_URL','https://storage.flutter-io.cn',User)
+   {% endcapture -%}
+{% else -%}
+   {% assign shell = 'your terminal' -%}
+   {% assign prompt = '$' -%}
+   {% assign comtoset = 'export ' -%}
+   {% assign installdirsuggestion = '`~/dev`' %}
+   {% capture envvarset -%}{{prompt}} {{comtoset}}{% endcapture -%}
+   {% capture setpath -%}{{envvarset}}PATH="$PWD/flutter/bin:$PATH"{% endcapture -%}
+   {% capture newdir -%}{{prompt}} mkdir ~/dev{% endcapture -%}
+   {% if id == 'macos' %}
+      {% capture unzip -%} {{prompt}} unzip{% endcapture -%}
+   {% else %}
+      {% capture unzip -%} {{prompt}} tar -xf{% endcapture -%}
+   {% endif %}
+   {% capture permaddexample -%}
+cat <<EOT >> ~/.zprofile
+{{envvarset}}PUB_HOSTED_URL="https://pub.flutter-io.cn"
+{{envvarset}}FLUTTER_STORAGE_BASE_URL="https://storage.flutter-io.cn"
+{{setpath}}
+EOT
+   {% endcapture -%}
+{% endif -%}
+{%- case id %}
+   {% when 'windows','macos' %}
+      {%- assign file-format = 'zip' %}
+      {%- assign download-os = id %}
+   {% when 'linux','chromeos' %}
+      {%- assign download-os = 'linux' %}
+      {%- assign file-format = 'tar.xz' %}
+{% endcase %}
+
+<div id="{{id}}" class="tab-pane
+  {%- if id == 'windows' %} active {% endif %}"
+  role="tabpanel" aria-labelledby="{{id}}-tab" markdown="1">
+
+This procedure requires using {{shell}}.
+
+1. Open a new window in {{shell}} to prepare running scripts.
+
+1. Set `PUB_HOSTED_URL` to your mirror site.
+
+   ```terminal
+   {{envvarset}}PUB_HOSTED_URL="https://pub.flutter-io.cn"
+   ```
+
+1. Set `FLUTTER_STORAGE_BASE_URL` to your mirror site.
+
+   ```terminal
+   {{envvarset}}FLUTTER_STORAGE_BASE_URL="https://storage.flutter-io.cn"
+   ```
+
+1. Download the Flutter archive from your mirror site.
+   In your preferred browser, go to
+   [Flutter SDK archive](https://flutter.cn/docs/release/archive?tab={{id}}).
+
+1. Create a folder where you can install Flutter. then change into it.
+
+   Consider a path like {{installdirsuggestion}}.
+
+   ```terminal
+   {{newdir}}; cd {{installdirsuggestion}}
+   ```
+
+1. Extract the the SDK from the {{file-format}} archive file.
+
+   This example assumes you downloaded the {{os}} version of the Flutter SDK.
+
+   ```terminal
+   {{unzip}} {{sdk | replace: "opsys", download-os}}{{file-format}}
+   ```
```

Comment:
I recommend adding an extra sentence to explicitly explain that they need to use the exact name of the specific bundle they downloaded here. When there are terminal code blocks, readers often just copy paste without much consideration, or they may be new to this type of activity and may not know.